src/js/components/file-input.js

```diff
@@ -330,6 +284,59 @@ const handleChange = (e, fileInputEl, instructions, dropTarget) => {
   }
 };
 
+/**
+ * When using an Accept attribute, invalid files will be hidden from
+ * file browser, but they can still be dragged to the input. This
+ * function prevents them from being dragged and removes error states
+ * when correct files are added.
+ * @param {event} e
+ * @param {HTMLElement} fileInputEl - file input element
+ * @param {HTMLElement} instructions - text to inform users to drag or select files
+ * @param {HTMLElement} dropTarget - target area div that encases the input
+ */
+const preventInvalidFiles = (e, fileInputEl, instructions, dropTarget) => {
+  const acceptedFilesAttr = fileInputEl.getAttribute("accept");
+  dropTarget.classList.remove(INVALID_FILE_CLASS);
+
+  // Runs if only specific files are accepted
+  if (acceptedFilesAttr) {
+    const acceptedFiles = acceptedFilesAttr.split(",");
+    const errorMessage = document.createElement("div");
+
+    // If multiple files are dragged, this iterates through them and look for any files that are not accepted.
+    let allFilesAllowed = true;
+    const scannedFiles = e.target.files || e.dataTransfer.files
+    for (let i = 0; i < scannedFiles.length; i += 1) {
+      const file = scannedFiles[i];
+      if (allFilesAllowed) {
+        for (let j = 0; j < acceptedFiles.length; j += 1) {
+          const fileType = acceptedFiles[j];
+          allFilesAllowed =
+            file.name.indexOf(fileType) > 0 ||
+            file.type.includes(fileType.replace(/\*/g, ""));
```

Comment:
This line seems to be failing in IE11 with `.txt` files.

```
Object doesn't support property or method 'includes'
file-input.js (317,11)
```

**PDF works**
![image](https://user-images.githubusercontent.com/3385219/118160534-099afe00-b3e4-11eb-8f88-8fd2cbb8ab9f.png)

**TXT doesn't**
![image](https://user-images.githubusercontent.com/3385219/118160647-32bb8e80-b3e4-11eb-8614-bdf25111c3ae.png)

![image](https://user-images.githubusercontent.com/3385219/118160604-1fa8be80-b3e4-11eb-9b8e-7daa01bbea77.png)

---

FWIW in [develop](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/develop/components/detail/file-input.html) on IE11 it allowed PDF/TXT, but also invalid files

![image](https://user-images.githubusercontent.com/3385219/118161015-b1183080-b3e4-11eb-81f7-fc9fac28754d.png)

Reply:
@thisisdano should we worry about this since IE11 is below 2%?

Reply:
I updated this with a helper function that could work for us, I don't have IE11 - @mejiaj is going to take a look when he get's a chance